.github/actions/templates/avm-validateModulePSRule/action.yml

```diff
@@ -37,6 +37,10 @@ inputs:
   psrulePath:
     description: "The path to PSRule configurations"
     required: false
+  psruleBaseline:
```

Comment:
Once the PSRule feature becomes available, unblocking this PR, I'd suggest to leverage the avm/utilities/tools/Invoke-WorkflowsForBranch.ps1 utility to trigger all module workflows. Enabling only static validation is enough. The utility will provide workflow badges as an output, that we can copy in the PR body to verify the reliability compliance for the whole library.

Reply:
Unresolved to get badges attached before the MR is merged

Reply:
done